README.md

```diff
@@ -328,6 +328,10 @@ npm install husky --save-dev
 
 Verify that your version of Git is `>=2.13.0`.
 
+#### Lerna Workspaces and Monorepos
```

Comment:
Seems like this fix would also address this happening in a Yarn Workspaces monorepo. Maybe we should call that out?

Reply:
Changed to `Lerna, Monorepos, and Yarn Worksapces` to cover everything. What do you think?

Reply:
Thanks :) Already updated the PR, what is your merge process?

Reply:
Maybe @typicode can chime in for merge process.

I'd love to test this out @bnbarak, what's the easiest way?

Reply:
#### First lets recreate the issue on master:
0. close this example of a basic Lerna monorepo https://github.com/reggi/lerna-tutorial
1.  Setup Husky
2. Run `yarn` inside `packages/usage`
3. Open `.git/hooks/huskey.local.sh`, expect to see `cd "packages/usage"` // <-- **this is the problem**

#### Test the fix
4. Checkout this PR in the example project
5. Set the env variable `HUSKY_RUN_FROM_ROOT="true"` or `HUSKY_RUN_FROM_ROOT="1"`
6. Remove `.git/hooks`
7. Run `yarn` inside `packages/usage`
8. Open `.git/hooks/huskey.local.sh`, expect to see `cd "."` // <-- **this is the fix**

Reply:
Piniging @lessen-inc and @typicode for their attention :)